### DISCUSSION TITLE
E7D81D0E-C7A5-450E-BD4A-4F3F00D2A416

### DISCUSSION THREAD

**Discussion by giscus** [celestiamobile/comments#16] [General]
# TOI-836

TOI-836 is an orange dwarf star located about 90 light-years away from the solar system. Two hot planets: a mini-Neptune (2.587 Earth Radii) and a super-Earth (1.704 Earth Radii). They were confirmed to be orbiting the star in 2022, Their masses have been observed (4.53 Earth Masses for TOI-836 b, 9.60 Earth Masses for TOI-836 c) and During observations of the system, TTVs (Transit Timing Variations) were observed during transits of TOI-836 c, but none on TOI-836 b. Leading to the possibility that there is a non-transiting third planet within the system.

https://celestia.mobi/addon?item=E7D81D0E-C7A5-450E-BD4A-4F3F00D2A416

**Comment by exocritic5457** [comment]
these planets seem to be put on top of the system and not replacing the others already there because there's 4 planets, 2 planets sharing 2 orbits each, after I downloaded this.

**Reply by Karistus** [reply]
Sorry this took so long, this should be fixed now
